docs/new_endpoint_tutorial.md

```diff
@@ -77,18 +77,18 @@ else if($source === 'fluview_meta') {
 ```
 
 Fortunately, the function `meta_fluview()` is already defined, so we can just
-reuse it. (It's used by the `meta` endpoint as mentioned above.) In general,
-you will likely need to define a new function named like
+reuse it. (It's used by the `meta` endpoint as mentioned above.) 
+You will likely need to define a new function named like
```

Comment:
Original wording emphasizes that this step is necessary for all new endpoints, not just this example

```suggestion
reuse it. (It's used by the `meta` endpoint as mentioned above.) In general,
you will likely need to define a new function named like
```